tst/EnergyPlus/unit/VariableSpeedCoils.unit.cc

```diff
@@ -7135,4 +7135,152 @@ TEST_F(EnergyPlusFixture, VariableSpeedCoils_ZeroRatedCoolingCapacity_Test)
+         "    -0.0000426316,           !- Coefficient5 y**2",
+         "    -0.0003364517,           !- Coefficient6 x*y",
+         "    13.89,                   !- Minimum Value of x",
+         "    22.22,                   !- Maximum Value of x",
+         "    12.78,                   !- Minimum Value of y",
+         "    51.67,                   !- Maximum Value of y",
+         "    0.7923,                  !- Minimum Curve Output",
+         "    1.2736,                  !- Maximum Curve Output",
+         "    Temperature,             !- Input Unit Type for X",
+         "    Temperature,             !- Input Unit Type for Y",
+         "    Dimensionless;           !- Output Unit Type",
+         ""});
+
+    ASSERT_TRUE(process_idf(idf_objects));
+
+    // Get coil inputs
+    VariableSpeedCoils::GetVarSpeedCoilInput(*state);
+
+    // Set input processing flag
+    state->dataVariableSpeedCoils->GetCoilsInputFlag = false;
+
+    // Setting predefined tables is needed though
+    OutputReportPredefined::SetPredefinedTables(*state);
+    // Set up some environmental parameters
+    state->dataEnvrn->OutDryBulbTemp = -5.0;
+    state->dataEnvrn->OutHumRat = 0.0009;
+    state->dataEnvrn->OutBaroPress = 99000.0;
+    state->dataEnvrn->OutWetBulbTemp =
+        Psychrometrics::PsyTwbFnTdbWPb(*state, state->dataEnvrn->OutDryBulbTemp, state->dataEnvrn->OutHumRat, state->dataEnvrn->OutBaroPress);
+    state->dataEnvrn->WindSpeed = 5.0;
+    state->dataEnvrn->WindDir = 270.0;
+    state->dataEnvrn->StdRhoAir = 1.1;
+
+    // Set coil parameters
+    int DXCoilNum = 1;
+    int const CyclingScheme = HVAC::ContFanCycCoil;
+
+    int constexpr SpeedCal = 1;
+    Real64 SpeedRatio = 0.2;
+
+    Real64 SensLoad = 1000.0;
+    Real64 LatentLoad = 0.0;
+    Real64 PartLoadFrac = 0.7;
+    Real64 OnOffAirFlowRatio = 1.0;
+
+    state->dataLoopNodes->Node(1).MassFlowRate = 0.2;
+    state->dataLoopNodes->Node(2).MassFlowRate = 0.2;
+    state->dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).AirMassFlowRate = 0.2;
+    state->dataEnvrn->OutDryBulbTemp = -5.0;
+
+    // Run a compressor "On" scenario first
+    HVAC::CompressorOperation CompressorOp = HVAC::CompressorOperation::On;
+    VariableSpeedCoils::SimVariableSpeedCoils(*state,
+                                              state->dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).Name,
+                                              DXCoilNum,
+                                              CyclingScheme,
+                                              CompressorOp, // compressor on/off. 0 = off; 1= on
+                                              PartLoadFrac,
+                                              SpeedCal,
+                                              SpeedRatio,
+                                              SensLoad,
+                                              LatentLoad,
+                                              OnOffAirFlowRatio);
+}
+
```

Comment:
What is the value of Evaporative Condenser Pump Rated Power Consumption after this call?